src/promptflow/promptflow/_cli/_pf/_connection.py

```diff
@@ -13,7 +13,7 @@
     add_param_yes,
```

Comment:
Add Changelog?

Reply:
This is not a user oriented feature, so should it not be added?